### ISSUE TITLE
Default shared memory limits too restrictive

### ISSUE THREAD

**Issue by mdales** [quantifyearth/shark#65]

**Comment by patricoferris** [comment]
Potentially relevant is where we set this in the obuilder (runc) world -- currently to `65536k` which is probably the same as docker: https://github.com/quantifyearth/obuilder/blob/850bd8bfd35370b6bdea4de9058804aed3c8b4e6/lib/sandbox.runc.ml#L202-L211

**Comment by mdales** [comment]
Yeah, I believe docker is the same, but 64MB is too small in a geospatial context. I added `parallel_save` to which shares data via shared memory, and really I think in this context we should set it to a large %age of actual memory. But if Obuilder is merged we can easily make that change for our use case.

**Comment by mdales** [comment]
Created this PR to raise it to 64GB for now. https://github.com/quantifyearth/obuilder/pull/2
